src/data/nat/enat.lean

```diff
@@ -128,7 +151,7 @@ then is_false $ λ h, hx $ dom_of_le_of_dom h hy
 else is_true ⟨λ h, (hy h).elim, λ h, (hy h).elim⟩
 
 /-- The coercion `ℕ → enat` preserves `0` and addition. -/
-def coe_hom : ℕ →+ enat := ⟨coe, enat.coe_zero, enat.coe_add⟩
+def coe_hom : ℕ →+ enat := ⟨coe, nat.cast_zero, nat.cast_add⟩
```

Comment:
*maybe* it's worth the effort to make this computable (untested):
```suggestion
def coe_hom : ℕ →+ enat := ⟨some, by simpa using nat.cast_zero, by simpa using nat.cast_add⟩
```
then again, if people have been relying on `coe_hom` being defeq to `coe`, then it's probably too much effort.

Reply:
I would then at least make it clear that this is not `coe`, despite the name.

Reply:
Well... `coe_coe_hom` claims that it *is* `coe`. If CI is happy with this PR, then I'm inclined to not spend further words on this distinction.

Reply:
Aahrg, this is breaking stuff that I really don't feel like fixing.